Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftRequestController.cs

```diff
@@ -415,158 +408,159 @@ internal async Task ProcessOpenShiftsRequests(string isRequestFromLogicApp)
-                            var entityToUpdate = await this.openShiftRequestMappingEntityProvider.GetOpenShiftRequestMappingEntityByKronosReqIdAsync(
-                                item.Id).ConfigureAwait(false);
+                        // Update the status in Azure table storage.
+                        var entityToUpdate = await this.openShiftRequestMappingEntityProvider.GetOpenShiftRequestMappingEntityByKronosReqIdAsync(
+                            item.Id).ConfigureAwait(false);
 
-                            if (entityToUpdate != null)
+                        if (entityToUpdate != null)
+                        {
+                            if (item.StatusName == ApiConstants.ApprovedStatus && entityToUpdate.ShiftsStatus == ApiConstants.Pending)
                             {
-                                if (item.StatusName == ApiConstants.ApprovedStatus && entityToUpdate.ShiftsStatus == ApiConstants.Pending)
-                                {
-                                    // Update the KronosStatus to Approved here.
-                                    entityToUpdate.KronosStatus = ApiConstants.ApprovedStatus;
+                                // Update the KronosStatus to Approved here.
+                                entityToUpdate.KronosStatus = ApiConstants.ApprovedStatus;
 
-                                    // Commit the change to the database.
-                                    await this.openShiftRequestMappingEntityProvider.SaveOrUpdateOpenShiftRequestMappingEntityAsync(entityToUpdate).ConfigureAwait(false);
+                                // Commit the change to the database.
+                                await this.openShiftRequestMappingEntityProvider.SaveOrUpdateOpenShiftRequestMappingEntityAsync(entityToUpdate).ConfigureAwait(false);
 
-                                    // Build the request to get the open shift from Graph.
-                                    var httpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
-                                    httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", allRequiredConfigurations.ShiftsAccessToken);
-                                    using (var getOpenShiftRequestMessage = new HttpRequestMessage(HttpMethod.Get, "teams/" + user.ShiftTeamId + "/schedule/openShifts/" + entityToUpdate.TeamsOpenShiftId))
+                                // Build the request to get the open shift from Graph.
+                                var httpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
+                                httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", allRequiredConfigurations.ShiftsAccessToken);
+                                using (var getOpenShiftRequestMessage = new HttpRequestMessage(HttpMethod.Get, "teams/" + user.ShiftTeamId + "/schedule/openShifts/" + entityToUpdate.TeamsOpenShiftId))
```

Comment:
a constant templated string would be nicer for this url

Reply:
Done